metaflow/runtime.py

```diff
@@ -610,6 +621,8 @@ def __init__(
             # easily. There is anyway a corresponding int id stored in the
             # metadata backend - so this should be fine.
             task_id = "control-%s-%s-%s" % (run, input_step, input_task)
+        else:
+            self.multinode_iterator = None
```

Comment:
is this used for something?

Reply:
Sorry, seems stray code from an earlier iteration (pun intended).